source/intro_release_notes/release_notes_enterprise/resolved_issues_661.rst

```diff
@@ -37,6 +37,7 @@ The following issues has been solved in 6.6.1:
 - `Fix filtered attributes for backup restoration (DEV_PREFIX, OS/UUID) <https://github.com/OpenNebula/one/issues/6044>`__.
 - `Fix filtering MAC attribute only when no_ip is used <https://github.com/OpenNebula/one/issues/6048>`__.
 - `Fix marketplace image download with wrong user from FireEdge Sunstone <https://github.com/OpenNebula/one/issues/6048>`__.
+- `Fix removed duplicate records in Machine type and CPU model inputs <https://github.com/OpenNebula/one/issues/6135>`__.
```

Comment:
I think it will be more understandable if it is:

```suggestion
- `Remove duplicated records in machine type and CPU model inputs <https://github.com/OpenNebula/one/issues/6135>`__.
```